app/views/idv/in_person/verify_info/show.html.erb

```diff
@@ -72,6 +105,11 @@ locals:
   </div>
   <div class="grid-row grid-gap grid-gap-2 padding-bottom-1 padding-top-1 border-y border-primary-light">
     <dl class='grid-col-fill margin-y-0'>
+      <% if @capture_secondary_id_enabled %>
+        <div class="padding-y-1">
+          <strong><%= t('headings.residential_address') %></strong>
+        </div>
+      <% end %>
       <div>
```

Comment:
Inside `_verify.html.erb` (template for /verify) there is an evaluation for `remote_identity_proofing`.
At the top of the file it has this comment `remote_identity_proofing - true if we're doing RIDP, false if we're doing IPP`.  Inside verify.html.erb we are passing in `remote_identity_proofing` set to false. It does not look like this route `idv/shared/verify` is being used else where so I don't believe I need this additional evaluation in this template. Let me know if you think otherwise.  I wanted to raise this as a red flag so it would be double checked  

<img width="816" alt="Screenshot 2023-05-09 at 9 32 10 AM" src="https://github.com/18F/identity-idp/assets/125507397/a5d5019a-72ab-40a2-9960-079642904946">

Reply:
Remote proofing verify_info_controller is using `app/views/idv/verify_info/show.html.erb`, so you can remove the checks in `_verify.html.erb`. You can run `spec/features/idv/doc_auth/verify_info_step_spec.rb` to confirm that all is working as expected.

Reply:
Agreed, you do not need to check `remote_identity_proofing` in this file. I created [LG-9760](https://cm-jira.usa.gov/browse/LG-9760) to track removing references to that variable from the FSM templates.